service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +40,57 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public AssessmentResult insertAssessmentResult(
+      UUID id, String assessmentResult, String veteranIcn, String diagnosticCode)
+      throws NoSuchFieldException {
+    ClaimEntity claimEntity =
+        claimRepository
+            .findById(id)
+            .orElseThrow(
+                () ->
+                    new NoSuchFieldException(
+                        "Could not match claim ID {" + id.toString() + "} in DB."));
+    Map<String, Object> evidence = new HashMap<>();
+    try {
+      evidence = objMapper.readValue(assessmentResult, Map.class);
```

Comment:
Can't you just map to AssesmentResult here? Since you have already defined it. It can simplify the following code quite a bit.  If you do that ICN and diagnostic code params become unnecessary.